common/metrics/config.go

```diff
@@ -50,22 +50,6 @@ type (
 		// - "milliseconds"
 		// - "bytes"
 		PerUnitHistogramBoundaries map[string][]float64 `yaml:"perUnitHistogramBoundaries"`
-
-		// Following configs are added for backwards compatibility when switching from tally to opentelemetry
-		// All configs should be set to true when using opentelemetry framework to have the same behavior as tally.
-
-		// WithoutUnitSuffix controls the additional of unit suffixes to metric names.
-		// This config only takes effect when using opentelemetry framework.
-		// Note: this config only takes effect when using prometheus via opentelemetry framework
-		WithoutUnitSuffix bool `yaml:"withoutUnitSuffix"`
-		// WithoutCounterSuffix controls the additional of _total suffixes to counter metric names.
-		// This config only takes effect when using opentelemetry framework.
-		// Note: this config only takes effect when using prometheus via opentelemetry framework
-		WithoutCounterSuffix bool `yaml:"withoutCounterSuffix"`
-		// RecordTimerInSeconds controls if Timer metric should be emitted as number of seconds
-		// (instead of milliseconds).
-		// This config only takes effect when using opentelemetry framework for both statsd and prometheus.
-		RecordTimerInSeconds bool `yaml:"recordTimerInSeconds"`
```

Comment:
SInce they are already released, I guess we can't simply removed it. Have to mark them deprecated for one release and remove in the one after I think. Also carryover the those values if set to those new ones under PrometheusConfig.

Reply:
Reverted